orca-front50/orca-front50.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
   testImplementation(project(":orca-pipelinetemplate"))
   testImplementation("com.github.ben-manes.caffeine:guava")
   testRuntimeOnly("net.bytebuddy:byte-buddy")
+  testImplementation("org.codehaus.groovy:groovy-json")
```

Comment:
nit ... please put with the other `testImplementation` entries, before `testRuntimeOnly`

Reply:
rebased with the edit